### PR TITLE
GSYE-188: Added a dictionary to map aggregated simulation result names to appropriate file names.

### PR DESCRIPTION
**Pull request by mrsaemir**
## Reason for the proposed changes
- In order to match the file names listed in the [Result Download](https://gridsingularity.github.io/gsy-e/results-download/) doc, and as requested by [GSYE-188](https://gridsingularity.atlassian.net/browse/GSYE-188)

## Proposed changes
- Added a dictionary to map aggregated simulation result names (keys) to appropriate file names.

**INTEGRATION_TESTS_BRANCH**=master
**GSY_FRAMEWORK_BRANCH**=master

### CODE REVIEW

File: src/gsy_e/gsy_e_core/export.py [hunk 1]
```diff
@@ -66,6 +66,26 @@
     3: "net_metering_pricing"
 }
 
+json_data_area_fields_mapping = {
```
Comment (hannesdiedrich) [inline src/gsy_e/gsy_e_core/export.py:69]: Minor, but I propose to rename this into:
`json_results_file_name_mapping`

**Review by hannesdiedrich — COMMENTED** [review]
One minor comment. LGTM in general.
Can you please take the opportunity to adapt the integration tests step that we have [here](https://github.com/gridsingularity/gsy-backend-integration-tests/blob/6f5b5cfcccf7e16708b1abb2ada6523ca3ca98cc/gsy_e_tests/integration_tests/steps/integration_tests.py#L480) that checks if the backend returns all expected file names?
Since we added this test, there were added some more. Can you please run a simulation locally and see which json files are expected and adapt the test step accordingly? Thanks a lot!

File: src/gsy_e/gsy_e_core/export.py (hunk 1 above)
Comment (mrsaemir) [reply src/gsy_e/gsy_e_core/export.py:69]: Actually, I used the exact name as gsy-web, so people can find about this one if they try searching for `json_data_area_fields_mapping` in GitHub (This is what I always do when I want to know which repo is using which other repo).
As, I didn't write `json_data_area_fields_mapping` in gsy-web, I preferred to not change it there, too!
But, still, if you think the name is misleading people, I can change it in gsy-e, maybe?

Comment (hannesdiedrich) [reply src/gsy_e/gsy_e_core/export.py:69]: Sorry, I did not see it in the gsy-web PR.
Actually, in gsy-web it is missleading, too. There I would rename it to e.g. `results_field_to_json_filename_mapping`. 
I do not have a strong opinion on exactly these names, but don't you agree that these (or a version of them) would be more explanatory? I agree, they should be the same in both repos. So please change them on both places, thanks a lot!

Comment (mrsaemir) [reply src/gsy_e/gsy_e_core/export.py:69]: You're right. both names are misleading. I will change the name and ask for another round of review.

**Review by hannesdiedrich — APPROVED** [review]
Perfect. LGTM

**Review by Hassan754 — APPROVED** [review]

**Pull request merged**